### ISSUE TITLE
Identify TTS High Value Assets

### ISSUE THREAD

**Issue by its-a-lisa** [18F/tts-tech-portfolio#270]
## Related Issues:

**Comment by afeld** [comment]
This is kind of done with selections of the Mission Essential Functions (MEFs) for our Continuity of Operation Plan (COOP), which are currently cloud.gov, login.gov, and usa.gov. More to do here?

**Comment by its-a-lisa** [comment]
@afeld identification of HVAs related to COOP is just _one_ of the necessary incorporations that need to be considered when considering HVA's the whole list from M-17-09 Management of Federal High Value Assets is...

> Incorporation of HVA activities into broader agency IT and information security and privacy  management planning activities, including:
-	  Enterprise risk management;
-	  Budget, procurement, and contract management plans to address potential assessor findings;
-	  Change management; 
-	  Information Security Continuous Monitoring (ISCM) Strategy;
-	  IT lifecycle management, including plans to upgrade legacy components, system migration, and disposal;
-	  Privacy compliance and Privacy Continuous Monitoring (PCM);5
-	  Performance measurement and metrics; and
-	  Contingency planning.

**Comment by its-a-lisa** [comment]
Plan to groom this card once the Private board is stood up; moving to waiting/blocked until that card is completed.

**Comment by dawnpm** [comment]
Flagging that the Search.gov *contract*, in its support of USA.gov's search function, has been considered essential in the past. Is that still the case?

**Comment by afeld** [comment]
> Plan to groom this card once the Private board is stood up

Latter part is done, though I don't consider the list of said assets to be sensitive information.

**Comment by its-a-lisa** [comment]
Yes, I think it is a widely debated opinion, but M-17-09 is pretty specific, so I would prefer to ere on the side of what the instructions say.
